sdk/common/src/main/java/io/opentelemetry/sdk/internal/ComponentRegistry.java

```diff
@@ -31,48 +33,25 @@ public ComponentRegistry(Function<InstrumentationScopeInfo, V> factory) {
     this.factory = factory;
   }
 
-  /**
-   * Returns the registered value associated with this name and {@code null} version if any,
-   * otherwise creates a new instance and associates it with the given name and {@code null} version
-   * and schemaUrl.
-   *
-   * @param instrumentationScopeName the name of the instrumentation scope.
-   * @return the registered value associated with this name and {@code null} version.
-   */
-  public V get(String instrumentationScopeName) {
-    return get(instrumentationScopeName, null);
-  }
-
-  /**
-   * Returns the registered value associated with this name and version if any, otherwise creates a
-   * new instance and associates it with the given name and version. The schemaUrl will be set to
-   * null.
-   *
-   * @param instrumentationScopeName the name of the instrumentation scope.
-   * @param instrumentationScopeVersion the version of the instrumentation scope.
-   * @return the registered value associated with this name and version.
-   */
-  public V get(String instrumentationScopeName, @Nullable String instrumentationScopeVersion) {
-    return get(instrumentationScopeName, instrumentationScopeVersion, null);
-  }
-
   /**
    * Returns the registered value associated with this name and version if any, otherwise creates a
    * new instance and associates it with the given name and version.
    *
    * @param instrumentationScopeName the name of the instrumentation scope.
    * @param instrumentationScopeVersion the version of the instrumentation scope.
    * @param schemaUrl the URL of the OpenTelemetry schema used by the instrumentation scope.
+   * @param attributes the attributes of the instrumentation scope.
    * @return the registered value associated with this name and version.
    * @since 1.4.0
```

Comment:
The addition of the attributes param makes it a new method and invalidates the `@since` right?
```suggestion
   * @since 1.18.0
```
Internal, so probably not terribly important.

Reply:
Yeah the `@since` is only required for methods part of the public api. I'll just remove the annotation altogether..